Add small note to README.md (#4556)
kyma-project/test-infra: prow/cluster/resources/external-secrets/README.md

      property: anotherKey # name of the field in the GCP Secret JSON, unused for plain values
```

### Note

> The trusted/untrusted file sare only applied to the trusted and untrusted clusters respctivly, while the workload file is applied to both types of clusters.

Reviewer: ```suggestion
The trusted/untrusted files are only applied to the trusted and untrusted clusters respectively, while the workload file is applied to both clusters.
```
